tests/server/test_config.py

```diff
@@ -38,3 +39,119 @@ def test_config_json(self):
         self.assertEqual(
             CONFIG._path, Path(__file__).parent.joinpath("config_test.json").resolve()
         )
+
+
+class TestDebug(unittest.TestCase):
+    """Test if debug mode is correctly set and respected"""
+
+    def test_env_variable(self):
+        """Set DEBUG environment variable and check CONFIG picks up on it correctly"""
+        os.environ["DEBUG"] = "1"
```

Comment:
I think the debug environment variable should be given a more optimade-specific name, e.g. `OPTIMADE_DEBUG`, especially if it's being exported.